server/core/src/https/errors.rs

```diff
@@ -40,30 +40,32 @@ impl IntoResponse for WebError {
                 (StatusCode::INTERNAL_SERVER_ERROR, inner).into_response()
             }
             WebError::OperationError(inner) => {
-                let (response_code, headers) = match &inner {
+                match &inner {
```

Comment:
The cache test error starts here. It's because you have (_, None, _) in the response, and that confuses the headers (I think).

But also, it looks like you're trying to stuff the error code as a string rather than just using the json version we have. Is there a reason to avoid the json here? I think you're trying to make it so the *server* sends the pretty error message here rather than an error code, when really the webui should be getting the error code and then renders the message from that.

Reply:
(Reverting this one segment btw fixes all the tests. )

Reply:
> But also, it looks like you're trying to stuff the error code as a string rather than just using the json version we have. Is there a reason to avoid the json here? I think you're trying to make it so the _server_ sends the pretty error message here rather than an error code, when really the webui should be getting the error code and then renders the message from that.

Previously it was running `serde_json::to_string` on the `OperationError`, and if that didn't work it'd just string-format it anyway, so we were already just dumping a bare string at the user and showing them that. I'm trying to make it *slightly* less bad for any client using the system without a rewrite of the whole error-handling on both the client/webui/CLI (until we work out how to do *that* rewrite) 😄

Reply:
Reverted it because ... downstreams are JSON-decoding the response, ew.